line-bot-model/src/main/java/com/linecorp/bot/model/message/imagemap/ImagemapExternalLink.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.bot.model.message.imagemap;
+
+import lombok.Value;
+
+/**
+ * labeled hyperlink displayed after the video is finished.
+ */
+@Value
+public class ImagemapExternalLink {
+    /**
+     * Webpage URL. Called when the label displayed after the video is tapped.
+     *
+     * <ul>
+     * <li>Max: 1000 characters</li>
+     * </ul>
+     */
+    String linkUri;
```

Comment:
Similar to @kazuki-ma 's suggestion at https://github.com/line/line-bot-sdk-java/pull/295#discussion_r236481890
How about making this to `URI`?

Reply:
done~